packages/snack-content/src/sdk.ts

```diff
@@ -133,3 +108,28 @@ export function standardizeDependencies(dependencies: any): SnackDependencies {
   }
   return result;
 }
+
+/**
+ * @internal
+ */
+function standardizePeerDependencies(peerDependencies: any): SnackDependencyVersions {
+  if (!peerDependencies) {
+    return peerDependencies;
+  }
+  let result: SnackDependencyVersions = peerDependencies;
+  for (const name in peerDependencies) {
+    const peerDep = peerDependencies[name];
+    if (typeof peerDep === 'string' || peerDep === null) {
+      // :thumbsup: regular peer-dependency
+    } else if (typeof peerDep === 'object' && typeof peerDep.version === 'string') {
+      result = result === peerDependencies ? { ...peerDependencies } : result;
+      result[name] = peerDep.version;
+    } else {
+      // Invalid peer-dependency
+      result = result === peerDependencies ? { ...peerDependencies } : result;
+      delete result[name];
+    }
+  }
+
+  return result;
+}
```

Comment:
No code changes here -- moved it below standardizeDependencies so that the more important and public functions come first, and implementation details come after.

Reply:
It's also not the easiest function to understand, thanks for leaving it intact 😄